content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -118,7 +118,8 @@ vcs_repo (map of keys)
 ## Namespace: cost_estimate
 
 The **cost_estimation namespace** contains data associated with the current run's cost estimate. This
-namespace is only present if cost a estimate is available.
+namespace is only present if a cost estimate is available. Note that cost estimates are never
```

Comment:
"never" feels a bit strong here. Could we switch it to just say "not"?

Reply:
Sure